sources/config.yaml

```diff
@@ -0,0 +1,12 @@
+sources:
+  - Inconsolata.glyphs
+axisOrder:
+  - wdth
+  - wght
+familyName: Inconsolata
+buildStatic: true
+flattenComponents: false
```

Comment:
@m4rc1e why false ? 
🔥 FAIL: Check glyphs do not have components which are themselves components. ([com.google.fonts/check/glyf_nested_components](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyf_nested_components))
🔥 FAIL The following glyphs have components which themselves are component glyphs:
Aacute
Abreve
uni1EAE
uni1EAE
uni1EB6
uni1EB6
uni1EB0
uni1EB0
uni1EB2
uni1EB4 and **333 more**.

Reply:
This family has been hinted with VTT. If we flatten these, the hinting won't be compatible. The version we currently serve also has this issue.